pollers/vacancies.js

```diff
@@ -55,7 +55,7 @@ function isEngageJobOrder(exteranId) {
 
 function timestampToDate(timestamp) {
 	const d = new Date(timestamp);
-	return d.getFullYear() + "-" + (d.getMonth() + 1) + "-" + d.getDate();
+	return d.toISOString().split("T")[0]; // we only need the date
```

Comment:
The issue here was that we would get a date like 2019-1-25 which is not a standard ISO local date like 2019-01-25 and it would cause deserialization issues.